Automatic Validation ended with:
> 2025-04-26T10:54:22.3558826Z ##[error] Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 74 File: NGWIN.PicPick.Beta.installer.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.installer.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 78 File: NGWIN.PicPick.Beta.locale.en-US.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.locale.en-US.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 71 File: NGWIN.PicPick.Beta.locale.zh-CN.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.locale.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.locale.zh-CN.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 72 File: NGWIN.PicPick.Beta.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.yaml
 2025-04-26T10:54:22.3576669Z ##[error] Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 74 File: NGWIN.PicPick.Beta.installer.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.installer.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 78 File: NGWIN.PicPick.Beta.locale.en-US.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.locale.en-US.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 71 File: NGWIN.PicPick.Beta.locale.zh-CN.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.locale.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.locale.zh-CN.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 72 File: NGWIN.PicPick.Beta.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.9.0.schema.json Line: 2, Column: 25 File: NGWIN.PicPick.Beta.yaml

(Automated response - build 1048.)

--- a/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.installer.yaml
+++ b/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.installer.yaml
@@ -1,0 +1,113 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: NGWIN.PicPick.Beta
+PackageVersion: 8.0.0-beta5
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- 3fr
+- all
+- arw
+- avcs
+- bay
+- bmp
+- cam
+- cr2
+- cr3
+- crw
+- cs1
+- cur
+- dc2
+- dcm
+- dcr
+- dcs
+- dcx
+- dds
+- dib
+- dic
+- dicom
+- dng
+- emf
+- erf
+- fax
+- fff
+- g3f
+- g3n
+- gif
+- hdp
+- heic
+- heics
+- heif
+- heifs
+- icb
+- ico
+- iev
+- iiq
+- j2c
+- j2k
+- jfif
+- jif
+- jp2
+- jpc
+- jpe
+- jpeg
+- jpg
+- jxr
+- k25
+- kdc
+- lyr
+- mdc
+- mef
+- mos
+- mrw
+- nef
+- nrw
+- orf
+- pbm
+- pcx
+- pdf
+- pef
+- pgm
+- pix
+- png
+- ppm
+- psb
+- psd
+- ptx
+- pxm
+- pxn
+- raf
+- raw
+- rdc
+- rle
+- rw2
+- rwl
+- sr2
+- srf
+- srw
+- targa
+- tga
+- tif
+- tiff
+- v2
+- vda
+- vst
+- wbmp
+- wdp
+- webp
+- wmf
+- x3f
+- xif
+ProductCode: PicPick
+ReleaseDate: 2025-03-03
+AppsAndFeaturesEntries:
+- ProductCode: PicPick
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.picpick.app/8.0.0-beta5/picpick_inst_x64.exe
+  InstallerSha256: 66A0EA77035BF38EEA294B4BAC80A7F3E121748240A397E6EFE7E58418A7E3DB
+ManifestType: installer
+ManifestVersion: 1.10.0

--- a/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.locale.en-US.yaml
+++ b/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.locale.en-US.yaml
@@ -15,7 +15,7 @@ LicenseUrl: https://picpick.app/eula/
 Copyright: Copyright© NGWIN All rights reserved.
 ShortDescription: All-in-one design tool for everyone
 Description: A full-featured screen capture and recording tool, intuitive image editor, color picker, color palette, pixel-ruler, protractor, crosshair, whiteboard and more.
-Moniker: picpick
+Moniker: picpick-beta
 Tags:
 - capture
 - color-picker

--- a/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.locale.en-US.yaml
+++ b/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.locale.en-US.yaml
@@ -1,0 +1,39 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: NGWIN.PicPick.Beta
+PackageVersion: 8.0.0-beta5
+PackageLocale: en-US
+Publisher: NGWIN
+PublisherUrl: https://picpick.app/
+PublisherSupportUrl: https://picpick.app/contact/
+PrivacyUrl: https://picpick.app/eula/
+PackageName: PicPick
+PackageUrl: https://picpick.app/
+License: Freeware
+LicenseUrl: https://picpick.app/eula/
+Copyright: Copyright© NGWIN All rights reserved.
+ShortDescription: All-in-one design tool for everyone
+Description: A full-featured screen capture and recording tool, intuitive image editor, color picker, color palette, pixel-ruler, protractor, crosshair, whiteboard and more.
+Moniker: picpick
+Tags:
+- capture
+- color-picker
+- image
+- picture
+- screenshot
+- scrolling-screenshot
+ReleaseNotes: |-
+  - Fixed an issue where the landing page was displayed after capturing.
+  - Fixed an issue where the window title was broken.
+  - Improved copying of selections to the clipboard so that the copied images include their layers.
+  - Fixed the issue where image scaling did not maintain its aspect ratio.
+  - Ensured that the pasted image’s position remains within the screen.
+  - Enhanced deletion and cutting of selections to allow for selective transparency.
+  - Fixed the issue that caused clipboard images to become distorted when pasted into external applications.
+  - Resolved the problem where the default text color was initialized as transparent.
+  - Fixed a bug that caused the canvas resizer to disappear after merging layers.
+  - Corrected the malfunction in the free-angle image rotation feature.
+  - Improved selection and shape-drawing tools to support mouse wheel scrolling and auto-scroll.
+ManifestType: defaultLocale
+ManifestVersion: 1.10.0

--- a/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.locale.zh-CN.yaml
+++ b/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.locale.zh-CN.yaml
@@ -1,0 +1,26 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.10.0.schema.json
+
+PackageIdentifier: NGWIN.PicPick.Beta
+PackageVersion: 8.0.0-beta5
+PackageLocale: zh-CN
+Publisher: NGWIN
+PublisherUrl: https://picpick.app/zh/
+PublisherSupportUrl: https://picpick.app/zh/contact/
+PrivacyUrl: https://picpick.app/zh/eula/
+PackageName: PicPick
+PackageUrl: https://picpick.app/zh/
+License: 免费软件
+LicenseUrl: https://picpick.app/zh/eula/
+Copyright: 版权所有© NGWIN，保留所有权利
+ShortDescription: 一款适合所有人的全能设计工具
+Description: 一款全功能的屏幕截图工具，具有直观的图像编辑器、拾色器、调色板、像素标尺、量角器、瞄准线和白板等。
+Tags:
+- 图像
+- 图片
+- 截图
+- 拾色器
+- 捕获
+- 滚动截图
+ManifestType: locale
+ManifestVersion: 1.10.0

--- a/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.yaml
+++ b/manifests/n/NGWIN/PicPick/Beta/8.0.0-beta5/NGWIN.PicPick.Beta.yaml
@@ -1,0 +1,8 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
+
+PackageIdentifier: NGWIN.PicPick.Beta
+PackageVersion: 8.0.0-beta5
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.10.0

